airbyte-integrations/connectors/source-qonto/source_qonto/spec.yaml

```diff
@@ -0,0 +1,28 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Qonto Spec
+  type: object
+  required:
+    - organization_slug
```

Comment:
@leilaballouard instead of slug you need to be able to support endpoint. So we can try with sandbox vs prod vs test endpoints. This needs to be supported before we can even test.

Reply:
Here I only added a "production" endpoint and a "mock server" endpoint, as it's documented in [Qonto's doc](https://api-doc.qonto.com/docs/business-api/6434cbb9d968d-qonto)